setup.py

```diff
@@ -103,6 +104,15 @@ def run(self):
         sys.exit(0 if errno == 0 else 1)
 
 
+class PostDevelop(develop):
+    """Post-installation for development mode."""
+    def run(self):
+        """Override develop.run."""
+        develop.run(self)
+        errno = os.system('ansible-galaxy collection install ansible.utils community.crypto')
```

Comment:
This command is also needed for production, right? So why call it develop mode? It looks a little ambiguous.

Reply:
move to `make postinstall`